test/logflare/rules_test.exs

```diff
@@ -34,17 +35,19 @@ defmodule Logflare.RulesTest do
                lql_string: "a:testing"
              })
 
-    # delete the rule
-    assert {:ok, _rule} = Rules.delete_rule(rule)
+    Rules.delete_rule!(rule.id)
```

Comment:
would prefer to refactor away the bang version as it is only used in one location and is not really necessary.